style.css

```diff
@@ -1018,12 +1016,16 @@ button:active {
 }
 
 @media screen and (max-width: 768px) {
```

Comment:
- [OPTIONAL] This is a project where we should follow a mobile first design so the desktop style should come after the mobile style. I would recommend you change the position of the code below this media query to above the media query you opened to style your mobile version so you don't need to have 2 media queries in your CSS file and follow the DRY best practices.